### PR TITLE
Updating CartoDB.js submodule pointing to last commit in develop branch

### PR DESCRIPTION
**Pull request by xavijam**
Basically adding this behaviour for the FullScreen button.
https://github.com/CartoDB/cartodb.js/pull/686

REVIEWER: @alonsogarciapablo 
Fixes #5705

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cartodb-ui",
-  "version": "3.19.48",
```
Comment (alonsogarciapablo) [inline package.json]: We should merge master into this branch and make sure this number is the right one.

**Comment by alonsogarciapablo** [comment]
LGTM!

**Comment by xavijam** [comment]
:D, thanks @alonsogarciapablo

**Comment by xavijam** [comment]
Frontend tests passed, fasten your seat belts :).

**Pull request merged**